examples/cache/Dockerfile-frontenvoy

```diff
@@ -1,7 +1,10 @@
 FROM envoyproxy/envoy-dev:latest
 
-RUN apt-get update && apt-get -q install -y \
-    curl
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y curl \
+    && apt-get autoremove -y && apt-get clean \
```

Comment:
im still not entirely convinced of the benefit of this - but if we are to keep it can you:

- put each command on its own line
- consolidate the `rm -rf` lines into one

Reply:
OK

Reply:
also - curl isnt needed in `cache`